Sources/SmileID/Classes/DocumentVerification/Model/OrchestratedDocumentVerificationViewModel.swift

```diff
@@ -118,23 +120,16 @@ internal class IOrchestratedDocumentVerificationViewModel<T, U: JobResult>: Obse
         DispatchQueue.main.async {
             self.step = .processing(.inProgress)
         }
-
-        let zip: Data
-        do {
-            let savedFiles = try LocalStorage.saveDocumentImages(
-                front: documentFrontFile,
-                back: documentBackFile,
-                selfie: selfieFile,
-                livenessImages: livenessFiles,
-                countryCode: countryCode,
-                documentType: documentType
-            )
-            let zipUrl = try LocalStorage.zipFiles(at: savedFiles.allFiles)
-            zip = try Data(contentsOf: zipUrl)
-            self.savedFiles = savedFiles
-        } catch {
-            print("Error saving document images: \(error)")
-            onError(error: SmileIDError.unknown("Error saving document images"))
+        guard let zip = saveDocumentImages(
+            front: documentFrontFile,
+            back: documentBackFile,
+            selfie: selfieFile,
+            livenessImages: livenessFiles,
+            countryCode: countryCode,
+            documentType: documentType,
+            jobId: jobId,
+            saveStage:.pending
+        ) else {
             return
```

Comment:
I would err on the side of making `saveDocumentImages` responsible for only saving images, and not for error handling. I think the error handling should be done here and `saveDocumentImages` should be marked as `throws`